src/components/Slider/Slider.tsx

```diff
@@ -2,14 +2,15 @@
 
 import React from 'react';
 
-import debounce from 'lodash/debounce';
-
+import {useControlledState} from '../../hooks';
```

Comment:
is it necessary here? BaseSlider component has out-of-the-box props for controlled and uncontrolled behaviour without this hook

Reply:
Yes, because we control state of the BaseSlider component.